puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_katello_core.yaml

```diff
@@ -0,0 +1,36 @@
+- job:
+    name: test_katello_core
+    project-type: matrix
+    logrotate:
+      daysToKeep: -1
+      numToKeep: 32
+    properties:
+      - github_katello
+    scm:
+      - katello
+      - foreman
```

Comment:
If we enable the MultipleSCMs plugin then for plugins we can use the SCM configuration to manage the base foreman and plugin repositories that are needed for running tests like this example here. This would require a small change proposed here (and not in #188) to clone foreman to a foreman directory. See:

https://github.com/theforeman/foreman-infra/pull/194/files#diff-cb6e769e57f35b144c8a27a6b3508b53R7
https://github.com/theforeman/foreman-infra/pull/194/files#diff-e1dc954418569c69c1e176c941c38c0fR3